docs/src/further_topics/ugrid/partner_packages.rst

```diff
@@ -0,0 +1,93 @@
+    * Link to docs if applicable (intersphinx?).
+
+Iris' UGRID Partner Packages
+****************************
+Python is an easy to use language and has formed a very strong collaborative
+scientific community, which is why Iris is written in Python. But it is not as
+computationally strong as other languages, and this is a problem for working
+with unstructured meshes where computations are higher in volume (more data)
+and complexity (less consistency). So there are operations - which Python
+can perform acceptably on a structured grids - but which would need too much
+time/resource to perform on an unstructured mesh. New approaches are needed...
+
+Python packages already exist that enable the use of powerful mesh analysis
+tools originally written in faster languages such as C. And the Iris team are
+working on wrapper solutions to 'bridge the gap' between these packages and
+Iris.
+
+These solutions have been placed in their own 'partner packages' for several
+reasons:
+
+* Can be useful to others who are not using Iris.
+
+  * Everyone working with multi-dimensional geographic datasets shares common
+    problems that need solving.
+  * Wider user base = stronger community = better solutions.
+
+* Only some Iris users will need them - they are **optional** Iris dependencies.
+
+  * They introduce a lot of new API.
+  * They introduce new large dependencies that take time to install and need
+    disk space.
+
+Below you can learn more about the partner packages and how they are useful.
+Specifics of what operations would require their installation can be found in:
+:doc:`operations`.
+
+
+`GeoVista`_
+===========
+.. image:: images/geovistalogo.svg
+   :width: 300
+   :class: no-scaled-link
+
+.. rubric:: "Cartographic rendering and mesh analytics powered by `PyVista`_"
+
+PyVista is described as "VTK for humans" - VTK is a very powerful toolkit for
+working with meshes, and PyVista brings that power into the Python ecosystem.
+GeoVista in turn makes it easy to use PyVista specifically for cartographic
+work, designed from the start with the Iris
+:class:`~iris.experimental.ugrid.Mesh` in mind.
+
+Applications
+------------
+* Interactively plot mesh data:
+
+  * On a 3D globe.
+  * On your favourite projection.
+
+* Extract a specific region from a mesh.
+* Combine multiple meshes into one.
+
+`iris-esmf-regrid`_
+===================
+.. image:: images/iris-esmf-regrid.svg
+   :width: 300
+   :class: no-scaled-link
+
+.. rubric:: "A collection of structured and unstructured ESMF regridding schemes for Iris"
+
+ESMF provide a sophisticated, performant regridding utility that supports a
+variety of regridding types with both structured grids and unstructured meshes.
+A Python interface does exist - ESMPy - however working with Iris
+:class:`~iris.cube.Cube`\s and :class:`~iris.experimental.ugrid.Mesh`\es is
+sub-optimal. iris-esmf-regrid provides easy Iris-ESMPy interoperability, as
```

Comment:
A minor point, but the use of the word sub-optimal here could suggest that the `Cube` and `Mesh` objects have inherent flaws which make working with them sub-optimal, or that ESMF has inherent shortcomings beyond not supporting dask and the API not knowing about iris.
This could be better phrased as something like:
"however, this is not designed to work directly with `Cube`s and `Mesh`es."

Reply:
Yes, I was a bit uncertain about this myself.

c670b92d1d6c1a89b32ed220c743300b4a9290f4